spacy_streamlit/visualizer.py

```diff
@@ -130,30 +133,57 @@ def visualize(
 
 
 def visualize_parser(
-    doc: spacy.tokens.Doc,
+    doc: Union[spacy.tokens.Doc, List[Dict[str, str]]],
     *,
     title: Optional[str] = "Dependency Parse & Part-of-speech tags",
     key: Optional[str] = None,
+    manual: bool = False,
+    displacy_options: Optional[Dict] = None,
```

Comment:
It would be nice, but I think at most we could do now is something like `Dict[str, Any]`. What I would really want is something like a `TypedDict` we could import here with the correct types, but it should probably come from `spacy.displacy` rather than within this library.